Api/DebugInterface.php

```diff
@@ -20,8 +20,8 @@
 interface DebugInterface
 {
 	/**
+	 * @return void
```

Comment:
nit: Can you have `@return` on the last line of the phpdoc? Matches the rest of the interfaces in this folder.

Reply:
fixed @matt-thomason